lib/phx_tools_web/live/phx_tools_live/index.ex

```diff
@@ -35,24 +35,15 @@ defmodule PhxToolsWeb.PhxToolsLive.Index do
     {:noreply,
      socket
      |> assign(seo_attributes: %{url: Endpoint.url() <> "/#{action}"})
-     |> assign(:source_code_url, source_code_url_from_os_or_live_action(action))}
+     |> assign(:source_code_url, source_code_url())}
```

Comment:
Actually, it's not dynamic anymore, so why not just hard-code in the template?

Reply:
Also look at the seo_attributes above.

Reply:
@mbashia, we don't need the `:source_code_url` assign anymore.
Hard-code it in the template.

Reply:
![Screenshot from 2024-11-21 14-52-24](https://github.com/user-attachments/assets/c32afc67-7c07-4acd-9607-5f74008ab071)
We can just hard code it here since it is passed into the components

Reply:
@almirsarajcic what do you think?

Reply:
Maybe not even here. When you refactor the `os_landing_card` component, maybe you'll put it inside.